src/lib/types/events.ts

```diff
@@ -309,6 +320,12 @@ export const IEventTypes = [
     PROJECT_ENVIRONMENT_REMOVED,
     DEFAULT_STRATEGY_UPDATED,
     SEGMENT_IMPORT,
+    INCOMING_WEBHOOK_CREATED,
+    INCOMING_WEBHOOK_UPDATED,
+    INCOMING_WEBHOOK_DELETED,
+    INCOMING_WEBHOOK_TOKEN_CREATED,
+    INCOMING_WEBHOOK_TOKEN_UPDATED,
```

Comment:
Do you see a case where the webhook can be updated? Maybe something like GH tokens where they have an expiration date and you can regenerate the secret?

Reply:
That's a good question. The way I see it:
 - You can update the incoming webhook by updating its enabled state (toggling them)
 - You can *potentially* update the token by updating its name

But no strong feelings about this. I'm also OK with not allowing updating the token names.

Reply:
Ok! I think I was not considering the token name as a property we could change